server/src/test/resources/backup.conf

```diff
@@ -0,0 +1,3 @@
+queues {
```

Comment:
a more indicative name of this file would be good :) it's a test file, right? not a backup of something?